server/cmwell-ws/app/security/AuthCache.scala

```diff
@@ -83,9 +100,12 @@ class AuthCache @Inject()(crudServiceFS: CRUDServiceFS)(implicit ec: ExecutionCo
 
   private def extractPayload(infoton: Infoton): Option[JsValue] = infoton match {
     case FileInfoton(_, _, _, _, _, Some(c), _) =>
-      Some(Json.parse(new String(c.data.get, "UTF-8")))
+      val jsValOpt = Try(Json.parse(new String(c.data.get, "UTF-8"))).toOption
```

Comment:
1. `Json.parse` also takes an `Array[Byte]`, So no need to `new String(...)`
2. if`c.data.get` might fail, we should catch it either in the pattern match case, or deal with `Try` failures, instead of hiding the exception, and logging (erroneously) that we got an invalid json. pattern match can be:
```
case FileInfoton(_, _, _, _, _, Some(FileContent(Some(c),_,_,_)), _) =>
```
Also, is it possible we get a `dataPointer` instead the data?
Shouldn't we handle such a case?

Reply:
1. Ok! I will remove the `new String`.
2. I'll add a case.
3. No, IRW is responsible for populating data in case when `dataPointer` is in use, and logically we will never get an Infoton without content. Also - in this case I really hope no UserInfoton will reach zStore threshold, LOL.

Reply:
Reg (3), then it shouldn't be leaked.
We should open another issue to fix `FileContent` class.

Reply:
ref #284